locales/en/allocations.json

```diff
@@ -36,6 +36,11 @@
       "heading": "People for allocation",
       "empty_slot": "Person to be allocated"
     },
+    "people_assign": {
+      "slots_empty": "{{slots_empty}} {{person_or_people}} to add",
```

Comment:
I think this content already exists above?

Reply:
Yep - but the content is different for the 2 types of user.  Checking up on this.